cmd/api/routers/routers.go

```diff
@@ -51,17 +53,17 @@ func (c *Controller) GetAllResources(context *gin.Context) {
 		return
 	}
 
-	resources, lastUUID, lastDate, err := c.Database.QueryResources(context.Request.Context(), kind, group, version, limit, uuid, date)
+	resources, lastUUID, lastDate, err := c.Database.QueryResources(context.Request.Context(), kind, apiVersion, limit, uuid, date)
 	if err != nil {
 		abort.Abort(context, err.Error(), http.StatusInternalServerError)
 		return
 	}
 
-	var continueToken string
-	if limit != "" {
-		// If the limit was specified, we create the token otherwise
-		// send an empty string
-		continueToken = pagination.CreateToken(lastUUID, lastDate)
+	continueToken := pagination.CreateToken(lastUUID, lastDate)
+	if limit == "" {
+		// When the limit is empty, we return an empty continueToken
+		// so there is no `continue` in the metadata answer
+		continueToken = ""
```

Comment:
This was the only different block, so I opted for changing it to the common one. However you have a point in calling `CreateToken` when its not needed.

Also you made me remember that now `limit` cant' be an empty string, since we are setting a default in `pagination.Middleware`. So I'm removing these conditionals and doing more changes related to `limit` not being an empty string.